denizen_scripts/survival/repo-link/cuboid_creator.dsc

```diff
@@ -53,7 +57,7 @@ cuboid_command:
       - if <context.args.get[2]||null> == null:
```

Comment:
checking if the list's size is equal to or greater than two would be better than checking if a tag fails indexing the second value in a list
```suggestion
      - if <context.args.size> == 2:
```